go.mod

```diff
@@ -101,6 +101,8 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
+require github.com/seancfoley/ipaddress-go v1.2.1
```

Comment:
give this to line 113 and remove indirect from there

Reply:
not sure why this happened, i reverted it now.